README.md

```diff
@@ -1 +1,119 @@
-# check-status 
+<!-- start title -->
+
+# gitgitgadget Action: check-status
+
+<!-- end title -->
+<!-- start description -->
+
+This is a GitHub action to perform various scheduled actions with projects using gitgitgadget to submit changes. It is used as a scheduled action running in a separate repo or the repo it is acting upon. The use case is for repos that will not allow complex GitHub actions in the codebase when the GitHub repo is a clone that is used to submit updates to a non-GitHub maintained repo.
+
+<!-- end description -->
+<!-- start contents -->
+<!-- end contents -->
+
+## Usage
+
+<!-- start usage -->
+
+```yaml
+- uses: gitgitgadget/check-status@v0.5.0
+  with:
+    # The action to be performed. It must be one of the following: "update-open-prs",
+    # "update-commit-mappings", "handle-open-prs", "handle-new-mails".
+    action: ""
+
+    # Repository owner.
+    repo-owner: ""
+
+    # Repository name.
+    repo-name: ""
```

Comment:
It's too bad that `${{ github.repository }}` includes the owner.

But maybe that's not too bad, maybe we should do the same, i.e. combine `repo-owner` and `repo-name` into a single `repository`?

Reply:
See previous comment.